conf/cassandra.yaml

```diff
@@ -2368,3 +2368,104 @@ drop_compact_storage_enabled: false
 #   compatibility mode would no longer toggle behaviors as when it was running in the UPGRADING mode.
 #
 storage_compatibility_mode: NONE
+
+# Configuration for AutoRepair Scheduler
+#auto_repair:
+#  # Enable/Disable auto repair scheduler.
+#  # If it is set to false, then the scheduler thread will not be spun up.
+#  # If it is set to true, then the repair scheduler thread will be created. The thread will
```

Comment:
Can you adjust the line endings to be more uniform?  Looking through the rest of cassandra.yaml, it looks like new lines start on around 120 character boundaries, and there some lines that end earlier (like here), and others that are longer than 120 (like 2393)

Reply:
Done